platformio.ini

```diff
@@ -101,6 +101,8 @@ extra_configs =
 ;default_envs = lolin_c3_mini
 ;default_envs = thingpulse-espgateway
 ;default_envs = thingpulse-espgateway-ethernet
+;default_envs = d1mini-temt6000-ls
```

Comment:
Just remove these 2 also and we will be good to go

Reply:
No problem, done.